components/webfield/AreaChairConsole.js

```diff
@@ -199,6 +203,9 @@ const AreaChairConsole = ({ appContext }) => {
         { accessToken }
       )
       const areaChairGroups = allGroups.filter((p) => p.id.endsWith(areaChairName))
+      const secondaryAreaChairGroups = secondaryAreaChairName
+        ? allGroups.filter((p) => p.id.includes(`/${secondaryAreaChairName}_`))
```

Comment:
I think we need to pass `Secondary_Area_Chairs` name instead of the singular version. 

A user could be added to the group but then removed and the anon group is not deleted. The source of truth should be VENUE_ID/Submission1/-/Secondary_Area_Chairs

Reply:
line 205 should be `const areaChairGroups = allGroups.filter((p) => p.id.endsWith('/' + areaChairName))`

and 206 can be `const secondaryAreaChairGroups = allGroups.filter((p) => p.id.endsWith('/' + secondaryAreaChairName))`

Reply:
i think / is not necessary? because it's checking endsWith

Reply:
Area_Chairs will match with Secondary_Area_Chairs, do we want that?